### PR TITLE
feat(api-v2): open-api transaction-service

### PR DESCRIPTION
**Pull request by pietro-tota**
### List of changes
Add version 2 for eCommerce transactions-service api for new transaction creation (POST transactions)

### Motivation and context
Introduce transactions-service transaction activation api version 2.
This new version is required in order to add NPG orderId mandatory field used during transaction activation avoiding modifications to the v1 api version (as it's used by production Checkout version)

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by Fernando-Granato — APPROVED** [review]
🚀

File: src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_base_policy.xml.tpl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<policies>
+    <inbound>
+      <set-header name="X-Client-Id" exists-action="delete" />
+      <!--> Add here some logic for value X-Client-Id header properly such as based on api key etc <-->
+      <base />
+      <set-backend-service base-url="https://${hostname}/pagopa-ecommerce-transactions-service/v2" />
```
Comment (infantesimone) [inline src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_base_policy.xml.tpl:6]: is v2 required?
if we add the `v2` in the openapi file I think we can avoid it here

Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_base_policy.xml.tpl:6]: Yes we can use both approach, I will change openapi descriptors :)

Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_base_policy.xml.tpl:6]: I tried modifying open api path (adding preceding v2) but this way v2 prefix will be duplicated, one from api version set in api module and one from openapi path so I would leave this as is. What do you think?

**Pull request merged**